storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -344,6 +348,12 @@
    :sync-retry (atom 0)
    :download-lock (Object.)
    :stormid->profiler-actions (atom {})
+   :cgroup-manager (if (conf STORM-CGROUP-ENABLE)
```

Comment:
Also, why do we only load the isolation plugin if cgroups is enabled? Maybe this config can go away and we can just check if `storm.resource.isolation.plugin` is set?